modules/virt-creating-multiple-storage-pools.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-multiple-storage-pools_{context}"]
+= Creating multiple storage pools
+
+If you have multiple available disks or partitions on your node and want to use all of them for the HPP, use any combination of `storagePools` with or without `pvcTemplates`. 
+
+.Procedure
+
+. Ensure that your nodes have an available blank disk or partition that can be discovered by the `LocalStorage` Operator (LSO).
```

Comment:
So technically the LSO is not needed, our primary use case does use the LSO, but anything that can create PVs will work. And I don't want to paint ourselves into saying you require LSO, since you don't. I would structure this as, here is how you use the pvcTemplate. And then below, as an addendum show how you can use the LSO to create PVs that the hpp can consume.

Reply:
Gotcha. I just added this note. I have the LSO Op page linked at the bottom of the assembly in Additional Resources.

[NOTE]
====
You can use the Local Storage Operator to create provisioned volumes that can used by the HPP.
====